src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.OpationStrategy.OperationHandler;
+import core.basesyntax.OpationStrategy.OperationStrategy;
+import core.basesyntax.OpationStrategy.OperationStrategyImpl;
+import core.basesyntax.Operations.*;
+import core.basesyntax.ShopServiceAndReportGenerator.ReportGenerator;
+import core.basesyntax.ShopServiceAndReportGenerator.ReportGeneratorImpl;
+import core.basesyntax.ShopServiceAndReportGenerator.ShopService;
```

Comment:
Similarly, the package name 'ShopServiceAndReportGenerator' might be incorrect. Ensure that the package name matches the actual directory structure and naming conventions used in your project.